### PR TITLE
Clean up remaining v1 terminology

### PR DESCRIPTION
**Pull request by DominikaK**
Cleans up remaining mentions of the /ez route and PlatformUI.

(2.0 and master)

### CODE REVIEW

File: docs/getting_started/ez_platform_features.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ Now you can check out the project structure and capabilities of the system.
 
 Once you have the Demo installed, take a look around it to familiarize yourself with the interface of eZ Platform.
 
-To access the back office of your installation, append `/ez` to its path: &lt;`your_installation path>/ez`. You will be prompted for the username and password, which by default are:
+To access the back office of your installation, append `/admin` to its path: &lt;`your_installation path>/admin`. You will be prompted for the username and password, which by default are:
```
Comment (MagdalenaZuba) [inline docs/getting_started/ez_platform_features.md]: Back Office

File: docs/getting_started/starting_ez_platform.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Starting eZ Platform
 
-Once you have your eZ Platform installed, all you need to do to start it is to run your Apache server. Then you can access your eZ Platform's front and back office. To access the UI backend, add `/ez` to your installation URL.
+Once you have your eZ Platform installed, all you need to do to start it is to run your Apache server. Then you can access your eZ Platform's front and back office. To access the UI backend, add `/admin` to your installation URL.
```
Comment (MagdalenaZuba) [inline docs/getting_started/starting_ez_platform.md]: as above

File: docs/getting_started/troubleshooting.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ If you're having difficulty completing installation on a system with limited RAM
 
 ## Upload Size Limit
 
-To make use of PlatformUI, you need to define the maximum upload sizeto be consistent with the maximum file size defined in the Content Type using a File, Media or Image Field Definition.
+To make use of the Back Office, you need to define the maximum upload sizeto be consistent with the maximum file size defined in the Content Type using a File, Media or Image Field Definition.
```
Comment (MagdalenaZuba) [inline docs/getting_started/troubleshooting.md]: size to be consistent

**Comment by DominikaK** [comment]
@MagdalenaZuba I agree with the comments, but they'll have to be introduced in all branches for consistency, and this PR is for v2 only. We'll have to fix this stuff when doing the audit of getting started

**Comment by MagdalenaZuba** [comment]
@DominikaK OK, let's add them to the 'todo' list

**Pull request merged**